api/app/signals/apps/signals/models/attachment.py

```diff
@@ -42,6 +42,7 @@ class Meta:
             ('sia_delete_attachment_of_parent_signal', 'Kan bijlage bij hoofdmelding verwijderen.'),
             ('sia_delete_attachment_of_child_signal',  'Kan bijlage bij deelmelding verwijderen.'),
             ('sia_delete_attachment_of_other_user', 'Kan bijlage bij melding van andere gebruiker verwijderen.'),
+            ('sia_delete_attachment_of_reporter', 'Kan bijlage toegevoegd door melder verwijderen.')
```

Comment:
Maybe rename this to `sia_delete_attachment_of_anonymous_user`. At this moment we assume that an attachment uploaded via the public endpoint is done by the reporter.

Reply:
And how is this different than the `sia_delete_attachment_of_other_user`? If this is to prevent users from deleting attachments uploaded by the reporter should we just not allow this at all?